.github/workflows/go_generate_update.yml

```diff
@@ -5,6 +5,12 @@ on:
     schedule:
         - cron:  '0 8-18/4 * * 1-5'
 
+env:
+    GIT_AUTHOR_NAME: Tugdual Saunier
+    GIT_COMMITTER_NAME: Tugdual Saunier
+    GIT_AUTHOR_EMAIL: tugdual.saunier@gmail.com
+    GIT_COMMITTER_EMAIL: tugdual.saunier@gmail.com
```

Comment:
I like the idea of having a clearly identified bot user.

Reply:
updated with the same account as before